### PR TITLE
Update Evaluate Accuracy function and Manage Mini batches efficiently

### PR DESCRIPTION
**Pull request by avinashsai**
*Issue #, if available:*
Issue: #1  acc.update(preds=predictions, labels=target)
                acc.get()[1]

 Managing mini batches efficiently using MXNet Gluon ArrayDataset and DataLoader Modules

*Description of changes:*
1. acc.update updates the accuracy value only for the current batch and acc.get returns the accuracy 
   score for the last batch and not for the entire train/test data.
  Hence I modified function that returns the accuracy score for the entire train/test data.

2. Sequential way of training mini batches will be slower. Also if the batch size if not a divisor of train/test 
   data size then those samples are ignored.
   Hence I used Gluon ArrayDataset loader which combines Ndarrays and forms a dataset object which 
   then be used to create Train/Test Mini batches using DataLoader module. Using last_batch='keep' 
   considers the left over mini batches if batch_size doesn't divide train/test data size

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by avinashsai** [comment]
Hi,
Can I know the status of this PR??
